### ISSUE TITLE
nbgrader: change the help link when using the "create assignment" toolbar from the upstream docs to our docs

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#4707]
Here's our docs:  https://doc.cocalc.com/howto/nbgrader.html

NOTE: Don't do this **yet** because our docs as I create this ticket have basically nothing in them.  Only change the link when our docs get written...

**Comment by DrXyzzy** [comment]
1. There is an nbgrader guide in the CoCalc docs in the teaching section: https://doc.cocalc.com/teaching-nbgrader.html . I'll link the howto page to it. 

1. The CoCalc nbgrader guide in the teaching docs does not duplicate upstream docs on the different cell types. I can add some content for that so that the create assignment toolbars for those cell types can link to the CoCalc menu.

**Comment by williamstein** [comment]
> The CoCalc nbgrader guide in the teaching docs does not duplicate upstream docs on the different cell types.

Yes, I think that would be very helpful, including a few screenshots.  

Thanks for the clarification regarding the link; I found the wrong link by just searching for "nbgrader" in the integrated docs search:

![2020-06-25 09_48_32-Search — CoCalc Manual documentation](https://user-images.githubusercontent.com/1276278/85763041-2db8f800-b6c9-11ea-95b8-ed1492372287.png)


We should somehow hack Sphinx and make this instead use Google site: search, since it **of course** does a much better job and gives the better link first:

![image](https://user-images.githubusercontent.com/1276278/85762994-2396f980-b6c9-11ea-82a4-72cb0e4e6b35.png)

**Comment by DrXyzzy** [comment]
I always use the index for searching the docs. I find it more helpful, probably because I maintain index entries. :-). So I didn't notice that the return from the integrated search at top does not go to the most helpful content.

**Comment by williamstein** [comment]
> I always use the index for searching the docs. 

I suspect the majority of people type into a box at the top to search for stuff in all possible settings these days (since trained by Google, etc., since birth).

**Comment by williamstein** [comment]
Hi @DrXyzzy to be clear, I'm changing the link in the sources, and you're improving the docs.

**Comment by williamstein** [comment]
Actually, better yet, I'll explain exactly where to make the change, and you can then easily do it. Edit this file and change the links in there to point to your docs instead of the official upstream docs... and that's it:

https://github.com/sagemathinc/cocalc/blob/master/src/smc-webapp/jupyter/nbgrader/cell-types.ts

In the meantime I'll add a new button somewhere that is just a link to our overall docs.  When you someday actually have docs with details that make sense there, update the links.
